spec/damerau_levenshtein_spec.js

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright (c) 2011, Chris Umbel
```

Comment:
About copyright:
It can increase projects size considerably.

Cannot we just add Chris Umbel to readme as contributor?
what do you think @reblws ?

Reply:
and remove the copyright from that file ?

Reply:
My apologies, I didn't catch this when I was pasting these tests onto a new file. I removed it. 

I do think it would be nice if there was a single file where copyright notices went so they aren't all over the place.